en/config/safety.md

```diff
@@ -239,24 +239,27 @@ Parameter | Description
 --- | ---
 [NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID) | Set the failsafe action: Disabled, Warn, Return mode, Land mode.
 
-### QuadChute Failsafe
+### Quad-chute Failsafe
 
 Failsafe for when a VTOL vehicle can no longer fly in fixed-wing mode, perhaps because a pusher motor, airspeed sensor or control surface failed.
-If triggered, the vehicle will immediately switch to multicopter mode.
-If the vehicle was in [Mission mode](../flight_modes/mission.md) it enters failsafe [Return mode](../flight_modes/return.md).
+If triggered, the vehicle will immediately switch to hover mode and execute action defined in parameter [COM_QC_ACT](../advanced_config/parameter_reference.md#COM_QC_ACT) .
```

Comment:
@sfuhrer I plan to merge all this, but note this change from hover mode to hold mode. I don't know what hover mode is, so if this is actually correct, then let me know and we can revert.
```suggestion
If triggered, the vehicle will immediately switch to Hold mode and execute action defined in parameter [COM_QC_ACT](#COM_QC_ACT) .
```

Reply:
yeah that hover was correct, Hold is wrong. We need to settle on a consistent naming - I usually refer to the VTOL states to "Hover" (instead of "MC") and "Aerodynamic forward flight" (instead of "FW"). For "Hover" we are quite well aligned in the team, though for how we want to call the fixed-wing/forwrad/cruise/aerodynamic flight we're not aligned. 
Also currently we mostly refer to them as MC and FW modes in the docs, so for now I guess we should keep to that.

Reply:
So we are absolutely consistent in the docs - Hold mode for position locked in MC and for path locked in FW. In MAVLink the mode is called cruise mode for FW I think, and we should perhaps use that.

Hover would be different, because to me that does not imply "locked in place". But up to you guys to decide and I can make it so. But it has to happen end to end and affect QGC too.

Reply:
No, for me "Hover" = MC. We also have VTOLs with only one one propeller, so they obviously can never be in "Multicopter" mode. 
For the change here: lets just change it to:
If triggered, the vehicle will immediately switch to **MC** mode and execute action defined in parameter [COM_QC_ACT](#COM_QC_ACT) .

Reply:
If you can't hold position in your MC I'd call it altitude hold mode: http://docs.px4.io/main/en/flight_modes/altitude_mc.html

I guess the point is what the user will see for the mode in QGC so they don't have a confusing perception.
Calling it MC mode here is fine though.

Can you make this and the other change?

Reply:
If not, ping me - might get to it before next week. Might now!